ui/start_ui.py

```diff
@@ -794,23 +852,24 @@ def set_rag_default_path(self, selector, rag_path):
 
     def _init_ui(self):
         mark_alive = None
+        private_key = paramiko.Ed25519Key.from_private_key_file("/root/.ssh/id_ed25519")
         for index in range(len(self.ray_nodes)):
             if "node:__internal_head__" in ray.nodes()[index]["Resources"]:
                 mark_alive = index
             node_ip = self.ray_nodes[index]["NodeName"]
             self.ssh_connect[index] = paramiko.SSHClient()
-            self.ssh_connect[index].load_system_host_keys()
-            self.ssh_connect[index].set_missing_host_key_policy(paramiko.RejectPolicy())
+            #self.ssh_connect[index].load_system_host_keys()
+            self.ssh_connect[index].set_missing_host_key_policy(paramiko.AutoAddPolicy())
```

Comment:
We can't use AutoAddPolicy because of [code scan issue](https://github.com/intel-sandbox/llm-ray/issues/76).

Reply:
@KepingYan , I see, thanks for the clarification, I'll make the change